examples/js/libs/inflate.min.js.map

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
It give an error but still works, right? I think it's because `inflate.min.js` has this on its code `//@ sourceMappingURL=inflate.min.js.map`